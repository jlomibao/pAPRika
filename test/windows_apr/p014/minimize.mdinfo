

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1129E+02     4.6890E-01     1.5987E+00     H34       107

 BOND    =        1.1117  ANGLE   =      107.7361  DIHED      =       51.4506
 VDWAALS =      -46.2650  EEL     =     1337.6058  EGB        =     -135.9011
 1-4 VDW =        5.2134  1-4 EEL =    -2132.2465  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================
