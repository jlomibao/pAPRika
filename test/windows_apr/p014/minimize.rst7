defa
  125
  -0.5217174   4.9759485   1.8352377   1.0157692   4.9108265   1.8351793
  -0.8854866   4.2790464   0.6084282  -0.8854010   4.2788620   3.0619657
  -0.9017042   6.0040775   1.8353272   1.4813410   5.9031381   1.8352378
   1.3195508   4.1855275   0.6086146   1.3196327   4.1853311   3.0616085
   2.6868404   4.0833023   0.1137348   2.6869508   4.0830871   3.5563975
   0.2013674   3.8582858   3.7829219   0.2012350   3.8586261  -0.1126831
   0.1802765   3.3484162   4.8901797   0.1800671   3.3489973  -1.2200381
   2.6632775   4.0469279  -0.9815127   3.2459590   4.9656080   0.4459431
   2.6634462   4.0465663   4.6516414   3.2460343   4.9654480   3.2242771
  -4.6359422   1.9609683   1.8353103  -3.8101048   3.2595261   1.8352698
  -4.2145813   1.2971521   0.6085762  -4.2145765   1.2971503   3.0620559
  -5.7162261   2.1465113   1.8353161  -4.4362553   4.1591788   1.8352868
  -3.0302827   3.1594605   0.6085065  -3.0302753   3.1594427   3.0620656
  -2.2565605   4.2914013   0.1135969  -2.2564182   4.2913054   3.5569429
  -3.3066080   2.0274409   3.7831147  -3.3066403   2.0274550  -0.1125156
  -2.8755116   1.7537585   4.8902295  -2.8756175   1.7538010  -1.2196516
  -2.2363178   4.2525440  -0.9816317  -2.7399732   5.2174682   0.4454878
  -2.2360452   4.2524011   4.6521675  -2.7398000   5.2174245   3.2251528
  -4.0834338  -3.1098885   1.8352157  -4.7950135  -1.7453537   1.8352683
  -3.2979426  -3.0769380   0.6084265  -3.2977698  -3.0769423   3.0618972
  -4.7843102  -3.9526290   1.8352599  -5.8872201  -1.8376801   1.8353149
  -4.3184049  -1.1200631   0.6084838  -4.3182821  -1.1200859   3.0620176
  -4.9113869   0.1162710   0.1137894  -4.9112935   0.1161965   3.5568182
  -3.4760938  -1.9254641   3.7829837  -3.4763418  -1.9254477  -0.1126119
  -3.0233909  -1.6889984   4.8900469  -3.0238105  -1.6889840  -1.2197347
  -4.8677573   0.1144642  -0.9814450  -5.9550196   0.1610879   0.4458457
  -4.8676138   0.1143155   4.6520505  -5.9549482   0.1609808   3.2248265
   0.5850508  -5.1654487   1.8348724  -0.9524972  -5.1002067   1.8349770
   0.9488596  -4.4684152   0.6081246   0.9490099  -4.4685202   3.0616348
   0.9649712  -6.1936024   1.8347973  -1.4181364  -6.0924866   1.8349751
  -1.2561280  -4.3748784   0.6082625  -1.2559695  -4.3749381   3.0617762
  -2.6234218  -4.2707424   0.1137523  -2.6232004  -4.2707680   3.5564633
  -0.1375554  -4.0477950   3.7827619  -0.1378076  -4.0476835  -0.1128461
  -0.1162182  -3.5375789   4.8898611  -0.1166194  -3.5374511  -1.2199304
  -2.6002177  -4.2340769  -0.9814932  -3.1837456  -5.1522384   0.4460793
  -2.5998493  -4.2341110   4.6517059  -3.1835894  -5.1522499   3.2242088
   4.6985335  -2.1490923   1.8347193   3.8734030  -3.4481045   1.8347247
   4.2767476  -1.4854189   0.6080489   4.2768456  -1.4854849   3.0614562
   5.7789156  -2.3340618   1.8346691   4.5000042  -4.3474433   1.8346786
   3.0934501  -3.3483786   0.6080057   3.0935466  -3.3484310   3.0615102
   2.3199361  -4.4805536   0.1132905   2.3201536  -4.4806662   3.5562632
   3.3693080  -2.2162805   3.7825524   3.3691912  -2.2162141  -0.1130233
   2.9381599  -1.9428478   4.8897073   2.9380236  -1.9427768  -1.2201591
   2.2996367  -4.4418261  -0.9819417   2.8035272  -5.4064917   0.4452807
   2.3000232  -4.4420296   4.6515017   2.8037191  -5.4065610   3.2241152
   4.1487915   2.9227317   1.8349397   4.8584080   1.5570350   1.8348314
   3.3640563   2.8905978   0.6075550   3.3641546   2.8904591   3.0623895
   4.8509949   3.7643668   1.8349538   5.9507489   1.6477584   1.8347741
   4.3807778   0.9320272   0.6082433   4.3809032   0.9319209   3.0614200
   4.9732112  -0.3044105   0.1130494   4.9733968  -0.3045398   3.5564573
   3.5392507   1.7378326   3.7823891   3.5391075   1.7380488  -0.1125877
   3.0850264   1.5009534   4.8887409   3.0848560   1.5013208  -1.2189493
   4.9289583  -0.3024306  -0.9821598   6.0170583  -0.3493611   0.4444131
   4.9292854  -0.3026370   4.6516724   6.0171980  -0.3494782   3.2249475
   0.0001530   0.0031079  14.7786685  -0.9139685  -0.5758214  14.5986027
   0.8629917  -0.6497816  14.5985526   0.0341711   0.8209818  14.0485210
   0.0234033   0.5608886  16.2148771   0.9308233   1.1613192  16.3537059
  -0.0231980  -0.5607890  17.2743052  -0.8310269   1.2345790  16.3537700
  -0.9305027  -1.1612177  17.1347348   0.8311950  -1.2343872  17.1347557
  -0.0000920  -0.0039894  18.7111670  -0.8629996   0.6488558  18.8911069
  -0.0341368  -0.8224576  19.4406529   0.9140119   0.5749857  18.8911570
  -0.0001077   0.0002465  -6.0041002  -0.0002058   0.0000523  -9.0000704
   0.0001678   2.2000035 -11.2000237
