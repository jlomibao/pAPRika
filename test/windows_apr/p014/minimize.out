
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:37:29

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/p014
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   614252 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

      
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

                                                                                
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -7.9697E+02     1.0074E+01     2.4799E+01     N3          8

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -45.6886  EEL     =     1326.5946  EGB        =     -129.5792
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -796.9711
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.1129E+02     4.6898E-01     1.5986E+00     H34       107

 BOND    =        1.1099  ANGLE   =      107.7352  DIHED      =       51.4527
 VDWAALS =      -46.2608  EEL     =     1337.5558  EGB        =     -135.8886
 1-4 VDW =        5.2111  1-4 EEL =    -2132.2107  RESTRAINT  =        0.0096
 EAMBER  =     -811.2953
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.1129E+02     4.6897E-01     1.5986E+00     H34       107

 BOND    =        1.1101  ANGLE   =      107.7353  DIHED      =       51.4525
 VDWAALS =      -46.2612  EEL     =     1337.5612  EGB        =     -135.8899
 1-4 VDW =        5.2114  1-4 EEL =    -2132.2146  RESTRAINT  =        0.0096
 EAMBER  =     -811.2953
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.1129E+02     4.6897E-01     1.5986E+00     H34       107

 BOND    =        1.1103  ANGLE   =      107.7354  DIHED      =       51.4522
 VDWAALS =      -46.2617  EEL     =     1337.5667  EGB        =     -135.8913
 1-4 VDW =        5.2116  1-4 EEL =    -2132.2184  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.1129E+02     4.6896E-01     1.5986E+00     H34       107

 BOND    =        1.1105  ANGLE   =      107.7355  DIHED      =       51.4520
 VDWAALS =      -46.2622  EEL     =     1337.5722  EGB        =     -135.8927
 1-4 VDW =        5.2119  1-4 EEL =    -2132.2224  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.1129E+02     4.6895E-01     1.5986E+00     H34       107

 BOND    =        1.1107  ANGLE   =      107.7356  DIHED      =       51.4518
 VDWAALS =      -46.2627  EEL     =     1337.5782  EGB        =     -135.8942
 1-4 VDW =        5.2121  1-4 EEL =    -2132.2268  RESTRAINT  =        0.0096
 EAMBER  =     -811.2953
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.1129E+02     4.6894E-01     1.5986E+00     H34       107

 BOND    =        1.1109  ANGLE   =      107.7357  DIHED      =       51.4515
 VDWAALS =      -46.2632  EEL     =     1337.5838  EGB        =     -135.8956
 1-4 VDW =        5.2124  1-4 EEL =    -2132.2307  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.1129E+02     4.6893E-01     1.5987E+00     H34       107

 BOND    =        1.1111  ANGLE   =      107.7358  DIHED      =       51.4513
 VDWAALS =      -46.2637  EEL     =     1337.5895  EGB        =     -135.8970
 1-4 VDW =        5.2127  1-4 EEL =    -2132.2349  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.1129E+02     4.6892E-01     1.5987E+00     H34       107

 BOND    =        1.1113  ANGLE   =      107.7359  DIHED      =       51.4511
 VDWAALS =      -46.2642  EEL     =     1337.5952  EGB        =     -135.8985
 1-4 VDW =        5.2129  1-4 EEL =    -2132.2389  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.1129E+02     4.6891E-01     1.5987E+00     H34       107

 BOND    =        1.1115  ANGLE   =      107.7360  DIHED      =       51.4508
 VDWAALS =      -46.2646  EEL     =     1337.6005  EGB        =     -135.8998
 1-4 VDW =        5.2132  1-4 EEL =    -2132.2427  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1129E+02     4.6890E-01     1.5987E+00     H34       107

 BOND    =        1.1117  ANGLE   =      107.7361  DIHED      =       51.4506
 VDWAALS =      -46.2650  EEL     =     1337.6058  EGB        =     -135.9011
 1-4 VDW =        5.2134  1-4 EEL =    -2132.2465  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.1129E+02     4.6890E-01     1.5987E+00     H34       107

 BOND    =        1.1117  ANGLE   =      107.7361  DIHED      =       51.4506
 VDWAALS =      -46.2650  EEL     =     1337.6058  EGB        =     -135.9011
 1-4 VDW =        5.2134  1-4 EEL =    -2132.2465  RESTRAINT  =        0.0096
 EAMBER  =     -811.2952
 NMR restraints: Bond =    0.008   Angle =     0.000   Torsion =     0.000
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.306    9.302    0.004    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.399  147.398    0.001    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.472   49.483    0.011    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   62.011   62.003    0.009    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.193 -146.178    0.015    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.551  -72.548    0.003    0.000 t
  DUM  DM1    3 --  C    BUT    2:   20.783   20.824    0.041    0.008 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.996  180.000    0.004    0.000 a
  DUM  DM1    3 --  C3   BUT    2:  179.889  180.000    0.111    0.000 a
                                       Total distance penalty:      0.008
                                       Total angle    penalty:      0.000
                                       Total torsion  penalty:      0.000
|                               RMS deviation from ideal bonds :      0.0094
|                               RMS deviation from ideal angles:      4.472
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.20 (21.33% of Gen B)
|                Calc gb diag               4.06 (39.34% of Gen B)
|                Calc gb off-diag           4.06 (39.29% of Gen B)
|                Other                      0.00 ( 0.04% of Gen B)
|             Gen Born time             10.32 (100.0% of Nonbo)
|          Nonbond force             10.32 (90.19% of Force)
|          Bond/Angle/Dihedral        1.08 ( 9.43% of Force)
|          Other                      0.04 ( 0.38% of Force)
|       Force time                11.45 (100.0% of Runmd)
|    Runmd Time                11.45 (99.80% of Total)
|    Other                      0.02 ( 0.19% of Total)
| Total time                11.47 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:37:29.610  on 04/10/2019
|           Setup done at 18:37:29.615  on 04/10/2019
|           Run   done at 18:37:41.079  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
